pkg/metrics/metrics.jsx

```diff
@@ -387,21 +390,50 @@ class CurrentMetrics extends React.Component {
                     const is_user = k.match(/^user.*user@\d+\.service.+/);
                     const label = k.replace(/.*\//, '').replace(/\.service$/, '');
                     // only keep cgroup basenames, and drop redundant .service suffix
-                    merged.push([label, v, is_user]);
+                    merged.push([label, v, is_user, false]);
+                }
+                // filter out podman containers
+                const matches = k.match(podmanSystemRe);
+                if (matches && v) {
+                    // truncate to 12 chars like the podman output
+                    const containerid = matches.groups.containerid.substr(0, 12);
+                    const is_user = k.match(/^user.slice/);
+                    merged.push([containerid, v, is_user, true]);
                 }
             });
             merged.sort((a, b) => b[1] - a[1]);
             return merged.slice(0, n);
         }
 
-        function serviceRow(name, value, is_user) {
-            const name_text = (
-                <Button variant="link" isInline component="a" key={name} onClick={ e => cockpit.jump("/system/services#/" + name + ".service" + (is_user ? "?owner=user" : "")) }>
+        function cgroupClickHandler(name, is_user, is_container) {
+            if (is_container) {
+                cockpit.jump("/podman");
+            } else {
+                cockpit.jump("/system/services#/" + name + ".service" + (is_user ? "?owner=user" : ""));
+            }
+        }
+
+        function serviceRow(name, value, is_user, is_container) {
+            const podman_installed = cockpit.manifests && cockpit.manifests.podman;
+            const original_name = name;
+            name = is_container ? _("pod") + " " + name : name;
```

Comment:
Renaming `name` is a bit confusing when jumping around with `*`/`#` and such. Perhaps leave it alone, and do a `const label = ...` instead?

Reply:
This is also better in the future as the cgroupClickHandler will get the container id instead of "pod $containerid"